### PR TITLE
MTP-2406 lenkeliste

### PR DESCRIPTION
**Pull request by petterkoding**

### CODE REVIEW

File: mt-kit/core/css/src/global/_links.scss [hunk 1]
```diff
@@ -138,20 +139,13 @@ $transportlink-colors: 'error' !default;
   }
 
   &.forward-arrow-end-link {
```
Comment (lha047) [inline mt-kit/core/css/src/global/_links.scss:141]: If we're not having any links with arrows after. Maybe remove the classes with "end" and update?

Comment (petterkoding) [reply mt-kit/core/css/src/global/_links.scss:141]: Figma has these examples though

Comment (lha047) [reply mt-kit/core/css/src/global/_links.scss:141]: ok

File: mt-kit/core/svelte/src/stories/Links.stories.svelte [hunk 2]
```diff
@@ -223,7 +223,7 @@
     <ul class="mt-ul m-t-xxs layout-grid list-unstyled">
       {#each cards as link}
         <li>
-          <Link href={link.href} class="forward-arrow-end-link" linkText={link.text} />
+          <Link href={link.href} class="forward-arrow" linkText={link.text} />
```
Comment (lha047) [inline mt-kit/core/svelte/src/stories/Links.stories.svelte:226]: Screenshot 2024-08-02 at 09 46 56
Shouldnt the hover effect be the slightly larger underline?

**Review by lha047 — CHANGES_REQUESTED** [review]
See comment

**Review by lha047 — APPROVED** [review]
👍 looks like it will be ok when the general link styling is applied

File: mt-kit/core/svelte/src/stories/Links.stories.svelte (hunk 2 above)
Comment (petterkoding) [reply mt-kit/core/svelte/src/stories/Links.stories.svelte:226]: MTP-2397 will fix it

**Pull request merged**